ansible_collections/arista/avd/molecule/eos_designs-mpls-isis-sr-ldp/inventory/group_vars/MPLS_TENANTS_NETWORKS.yml

```diff
@@ -7,6 +7,13 @@ tenants:
       # Port-based vpws service
       - name: TEN_A_site2_site5_eline_port_based
         type: vpws-pseudowire
+        subinterfaces:
```

Comment:
Adding the subinterfaces here may affect test coverage for a port-based vpws service (as the comment says above). Maybe just add a new service or find one which already has subinterfaces. Thanks.

Reply:
I added a different service for the subinterfaces. Also i renamed this one with correct site names.